src/common/types.cpp

```diff
@@ -114,13 +150,13 @@ std::string Ip6Prefix::ToString() const
     return strBuilder.str();
 }
 
-std::string MacAddress::ToString(void) const
+std::string Ip6NetworkPrefix::ToString() const
 {
-    char strbuf[sizeof(m8) * 3];
+    char strbuf[INET6_ADDRSTRLEN];
 
-    snprintf(strbuf, sizeof(strbuf), "%02x:%02x:%02x:%02x:%02x:%02x", m8[0], m8[1], m8[2], m8[3], m8[4], m8[5]);
+    snprintf(strbuf, sizeof(strbuf), "%x:%x:%x:%x::0/64", htobe16(m16[0]), htobe16(m16[1]), htobe16(m16[2]),
```

Comment:
```suggestion
    snprintf(strbuf, sizeof(strbuf), "%x:%x:%x:%x::/64", htobe16(m16[0]), htobe16(m16[1]), htobe16(m16[2]),
```

Reply:
Btw, this method only supports prefixes of length 64. Do we need to support other lengths?

Reply:
The length of the `otIp6NetworkPrefix ` is a fixed length 64. So this class only supports 64 bits prefix.